### ISSUE TITLE
Props Change Event Not Working in Datepicker Component

### ISSUE THREAD

**Issue by MhinHub** [themesberg/flowbite-react#990]
### Discussed in https://github.com/themesberg/flowbite-react/discussions/977

<div type='discussions-op-text'>

<sup>Originally posted by **MhinHub** September 21, 2023</sup>
I am having an issue getting the selected value from the [`Datepicker`](https://www.flowbite-react.com/docs/components/datepicker) component from the `flowbite-react` library. Using with NextJS.
<br/>
The component displaying nicely.
<br/>
I have tried the following code, but return nothing when the date is selected:

```tsx
import { Datepicker } from "flowbite-react";

export default function ExampleComp(){
 return (
   <Datepicker
       onChange={(e) => console.log(e)} // not working
       onInput={(e) => console.log(e)} // not working
       onSelectedDateChanged={(e) => console.log(e)} // Error: Does not exist on type 'IntrinsicAttributes & DatepickerProps'
      />
  )
}


```

Nothing specific use case in the official documentation page but in [`storybook`](https://storybook.flowbite-react.com/?path=/story/components-datepicker--default) there is prop `onSelectedDateChanged`. I'm using it but `Error: Does not exist on type 'IntrinsicAttributes & DatepickerProps`
<br/>
Kindly help.
<br/>

version
- "flowbite": "^1.8.1",
- "flowbite-react": "^0.6.0",
- "tailwindcss": "3.3.3",
- "next": "13.4.19"

[See on Stackoverflow?](https://stackoverflow.com/q/77146513/15409068)</div>

**Comment by rluders** [comment]
@MhinHub is it still happening? I noticed that you opened the issue on the day that we released the `0.6.1`, at the same time your issue was originally posted on Sep 21st, and the `Datepicker` component had a but at `< 0.6.1` versions. 

Here is the commit: https://github.com/themesberg/flowbite-react/commit/fe9f63f0cdb04f5ed64baa9ff8755182ef6c8fce

You should use `onSelectedDateChanged` to pick up the value. I'll wait for your confirmation.

Please, be aware that you need to update your version to `0.6.1` otherwise it would not work.

**Comment by hknowlton** [comment]
I was having the same issue, upgrading to 0.6.1, and now I am seeing this issue -

<img width="652" alt="image" src="https://github.com/themesberg/flowbite-react/assets/23346903/d3b7efd7-dc64-4b2f-bdc5-d0f926490615">

I have uninstalled and reinstalled node modules. Also tried v0.6.2

**Comment by MhinHub** [comment]
> @MhinHub is it still happening? I noticed that you opened the issue on the day that we released the `0.6.1`, at the same time your issue was originally posted on Sep 21st, and the `Datepicker` component had a but at `< 0.6.1` versions.
> 
> Here is the commit: [fe9f63f](https://github.com/themesberg/flowbite-react/commit/fe9f63f0cdb04f5ed64baa9ff8755182ef6c8fce)
> 
> You should use `onSelectedDateChanged` to pick up the value. I'll wait for your confirmation.
> 
> Please, be aware that you need to update your version to `0.6.1` otherwise it would not work.

I've try to update version to `0.6.1` even `0.6.2` and I'm seeing the issue same with:
 @hknowlton  https://github.com/themesberg/flowbite-react/issues/990#issuecomment-1741371791

```
Module not found: Can't resolve '~/src/components/FloatingLabel/theme'
```

**Comment by tulup-conner** [comment]
I can't reproduce this, and the most recent comment references an issue that's been resolved for a while. Let me know if you do still have problems here.
